### DISCUSSION TITLE
crate

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3220] [Q&A]
I use SEI like this, 
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/bd1abe9e-1b69-4314-83b3-98f089672e51)

but why do I get the number of cycles when it reaches termination = 80%, that is, the higher the charge and discharge rate (CRate) the more cycles:
0.1C = 636 cycles
0.2C = 1121 cycles
0.3C = 1441 cycles
1C = 1959 cycles
2C = 1990 cycles
3C = 1972 cycles

Is that true? @tinosulzer @brosaplanella

**Comment by valentinsulzer** [comment]
Yes. The SEI models are basically just calendar aging, and the higher C-rates have shorter cycles so less degradation per cycle. This is also backed up by experiments e.g. https://iopscience.iop.org/article/10.1149/2.0231904jes/pdf Try plotting capacity vs time or Ah-throughput.

**Reply by zeecode07** [reply]
but the number of cycles generated when I tried the simulation, the higher the C rate, the higher the number of cycles. like this:
0.1C = 636 cycles
0.2C = 1121 cycles
0.3C = 1441 cycles
1C = 1959 cycles
2C = 1990 cycles
3C = 1972 cycles
is there something wrong with my code? this is example for 3C:
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/c233acc9-77b9-4126-ad87-6395e146b239)

i don't know why Crate is getting higher, cycle count is also getting higher? 
please i need your help sir🙏🏻🙏🏻 

@tinosulzer @brosaplanella @rtimms

**Reply by valentinsulzer** [reply]
At higher C-rates cycles are shorter so there are more cycles in the same amount of time. Time (not cycle count) is what affects SEI growth.

**Reply by zeecode07** [reply]
what is the formula to find battery lifetime based on Crate and number of cycles? @tinosulzer

**Reply by brosaplanella** [reply]
There is no such formula as it is a very complex process.

**Reply by zeecode07** [reply]
can it be calculated by 
lifetime = cycle number / charge and discharge rate

**Reply by brosaplanella** [reply]
No, lifetime is a product of quite complex processes so it can only be predicted via simulations of such processes.

**Reply by zeecode07** [reply]
then how to predict lifetime?

**Reply by brosaplanella** [reply]
As I said in my previous comment, you would need to run a simulation to predict the lifetime of the battery.

**Reply by zeecode07** [reply]
but the time to reach the end of the cycle when C rates are high the time is longer so the number of cycles is large than the C rates are low the time is faster and the number of cycles is small @tinosulzer

**Comment by zeecode07** [comment]
> At higher C-rates cycles are shorter so there are more cycles in the same amount of time. Time (not cycle count) is what affects SEI growth.

but the time to reach the end of the cycle when C rates are high the time is longer so the number of cycles is large than the C rates are low the time is faster and the number of cycles is small @tinosulzer 

my code when simulating experiment like this, this is when 0.1C :
experiment = pybamm.Experiment([
    ("Discharge at 0.1C until 2.5V",
     "Rest for 1 hour",
     "Charge at 0.1C until 4.2V", 
     "Hold at 4.2V until C/50")
] * 5000,
termination="80% capacity")

sim = pybamm.Simulation(model, experiment=experiment, parameter_values=parameter_values, var_pts=var_pts)
sol = sim.solve()

**Reply by zeecode07** [reply]
can you help me sir why this happen? @tinosulzer @brosaplanella

**Reply by brosaplanella** [reply]
At higher C-rates the discharge time should be shorter than expected, so the total capacity you can extract is smaller. Here is a hypothetical example: if a battery discharges at 1C in exactly 1 hour, at 2C it would discharge in a bit less than half an hour and thus the total discharge capacity would be lower than at 1C. These effects add on when doing many cycles, and that's what we suspect causes the lower ageing at higher C-rates.

About the second point, I don't think I understand your question. If you run that simulation and then extract the number of cycles until 80% capacity that would give you a measure of the lifetime. But there is no analytical formula that would allow you to compute this, so you need to use a simulation.

**Comment by Naveen-S06** [comment]
Can anyone kindly help how to retrieve cycle number? I don't see that being an output variable in the default parameter set. I'm new to PyBamm and it would be great if someone could help me in this
